### DISCUSSION TITLE
async driver for SQL Server

### DISCUSSION THREAD

**Discussion by techvslife** [sqlalchemy/sqlalchemy#6525] [Usage Questions]
Is there a suggested async driver to use for MS SQL Server. Thank you.

EDIT: I see that is a reported issue:
https://github.com/sqlalchemy/sqlalchemy/issues/6521

**Comment by CaselIT** [comment]
As mentioned in the linked issue, there is currently no support for an async driver for mssql. It's on our radar, but we don't have an eta for it at the moment

**Reply by techvslife** [reply]
Thank you, it would be helpful to have an async driver for SQL Server, as I can't test the new sqlalchemy features without such a driver. (Perhaps Microsoft could write a driver.)  

I checked out the "aioodbc" driver package (since ODBC should work with SQL Server), but even if aioodbc does work with sqlalchemy and SQL Server, it hasn't been updated in years, so I'm not confident about using it, e.g.:
https://github.com/aio-libs/aioodbc/issues/245

**Reply by CaselIT** [reply]
Yes, I think the lack of a popular and maintained async library for mssql (and oracle) is why in part they don't yet have asyncio support. hopefully that will change someday

**Comment by CaselIT** [comment]
support for it is now released in 2.0.23
